src/modules/scraper/service.ts

```diff
@@ -281,4 +283,23 @@ export class ScraperService {
       });
     }
   }
+
+  public async backfillFilledDate(body: BackfillFilledDateBody) {
+    const deposits = await this.depositRepository
+      .createQueryBuilder("d")
+      .andWhere("d.filledDate is null")
+      .andWhere("d.status='filled'")
+      .andWhere("d.depositDate is not null")
+      .orderBy("d.id", "DESC")
+      .take(body.count || 1000)
+      .getMany();
+    this.logger.debug(`[backfillFilledDate] found ${deposits.length} deposits`);
+
+    for (const deposit of deposits) {
+      this.logger.debug(`[backfillFilledDate] get filled date for ${deposit.id}`);
```

Comment:
This logging call is unbounded by the code. I'm not sure this is a good idea to log each one.

Reply:
Removed!